### PR TITLE
switch actions to use new output format

### PR DESCRIPTION
**Pull request by dlockhart**

### CODE REVIEW

**Review by devpow112 — APPROVED** [review]

File: incremental-release/action.yml [hunk 1]
```diff
@@ -54,7 +54,7 @@ runs:
         else
           echo "Did not find commit message flag, using DEFAULT_INCREMENT (${{ inputs.DEFAULT_INCREMENT }})"
         fi
-        echo "::set-output name=increment::$(echo "$TYPE")"
+        echo "increment=$(echo "$TYPE")" >> $GITHUB_OUTPUT
```
Comment (eKoopmans) [inline incremental-release/action.yml:57]: Same comment about re-echoing - I think now's a good time to remove these. Can't think of any reason why to keep it (except for the `$(cat .lmsrelease)` one, which is doing something different).

File: semantic-release/action.yml [hunk 2]
```diff
@@ -74,7 +74,7 @@ runs:
         GITHUB_TOKEN: ${{ inputs.GITHUB_TOKEN }}
         NPM_TOKEN: ${{ inputs.NPM_TOKEN }}
       run: |
-        echo "::set-output name=version::"
+        echo "version::=" >> $GITHUB_OUTPUT
```
Comment (eKoopmans) [inline semantic-release/action.yml:77]: Missed the `::` removal here.

File: update-package-lock/action.yml [hunk 3]
```diff
@@ -52,7 +52,7 @@ runs:
         id: check-for-changes
         run: |
           echo -e "\n\e[34mChecking For Changes"
-          git diff --exit-code package-lock.json && echo "::set-output name=changes-exist::$(echo false)" || echo "::set-output name=changes-exist::true"
+          git diff --exit-code package-lock.json && echo "changes-exist=$(echo false)" >> $GITHUB_OUTPUT || echo "changes-exist=true" >> $GITHUB_OUTPUT
```
Comment (devpow112) [inline update-package-lock/action.yml:55]: Same double echoing of stuff here

**Comment by devpow112** [comment]
There's a bunch of `echo $(echo $VARIABLE)` type of stuff. All of it was pre-existing. Maybe can be cleaned up in another pass.

File: incremental-release/action.yml (hunk 1 above)
Comment (dlockhart) [reply incremental-release/action.yml]: Ok I've cleaned these up too!

**Review by eKoopmans — APPROVED** [review]

**Review by devpow112 — APPROVED** [review]

**Pull request merged**